src/renderer/components/hotbar/hotbar-add-command.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { observer } from "mobx-react";
+import { hotbarStore } from "../../../common/hotbar-store";
+import { CommandOverlay } from "../command-palette";
+import { Input, InputValidator } from "../input";
+
+const uniqueWorkspaceName: InputValidator = {
```

Comment:
I don't think this should be called ...workspace...

Reply:
Good catch, fixed.